### PR TITLE
Benchmark modified for tf change

### PR DESCRIPTION
**Pull request by miaecle**
Some minor changes, including the import of weight_init_stddevs and bias_init_consts

### CODE REVIEW

File: examples/benchmark.py [hunk 1]
```diff
@@ -184,95 +190,121 @@ def benchmark_train_and_valid(base_dir, train_dataset, valid_dataset, tasks,
...
-          len(tasks), n_features, learning_rate=learning_rate,
-          layer_sizes=layer_sizes, dropouts=dropouts, batch_size=batch_size, 
-          seed=seed, verbosity=verbosity)
+    # Building tensorflow MultiTaskDNN model
+    model_tf = dc.models.TensorflowMultiTaskClassifier(len(tasks),
+        n_features, layer_sizes=layer_sizes, 
+        weight_init_stddevs=weight_init_stddevs,
+        bias_init_consts=bias_init_consts, dropouts=dropouts, penalty=penalty, 
+        penalty_type=penalty_type, batch_size=batch_size, 
+        learning_rate=learning_rate, verbosity=verbosity)
```
Comment (rbharath) [inline examples/benchmark.py]: Can you add `seed=seed` here?

File: examples/benchmark.py [hunk 2]
```diff
@@ -184,95 +190,121 @@ def benchmark_train_and_valid(base_dir, train_dataset, valid_dataset, tasks,
...
+    model_robust = dc.models.RobustMultitaskClassifier(len(tasks),
+        n_features, layer_sizes=layer_sizes, 
+        weight_init_stddevs=weight_init_stddevs,
+        bias_init_consts=bias_init_consts, dropouts=dropouts,
+        bypass_layer_sizes=bypass_layer_sizes,
+        bypass_weight_init_stddevs=bypass_weight_init_stddevs,
+        bypass_bias_init_consts=bypass_bias_init_consts,
+        bypass_dropouts=bypass_dropouts, penalty=penalty, 
+        penalty_type=penalty_type, batch_size=batch_size,
+        learning_rate=learning_rate, verbosity=verbosity)
```
Comment (rbharath) [inline examples/benchmark.py]: Can you add `seed=seed` here?

File: examples/benchmark.py [hunk 3]
```diff
@@ -184,95 +190,121 @@ def benchmark_train_and_valid(base_dir, train_dataset, valid_dataset, tasks,
...
-    model_logreg = dc.models.TensorflowLogisticRegression(
-          len(tasks), n_features, learning_rate=learning_rate, penalty=penalty, 
-          penalty_type=penalty_type, batch_size=batch_size, 
-          seed=seed, verbosity=verbosity)
- 
+    # Building tensorflow logistic regression model
+    model_logreg = dc.models.TensorflowLogisticRegression(len(tasks),
+        n_features, penalty=penalty, penalty_type=penalty_type, 
+        batch_size=batch_size, learning_rate=learning_rate, 
+        verbosity=verbosity)
```
Comment (rbharath) [inline examples/benchmark.py]: Same with `seed` here.

File: examples/benchmark.py [hunk 4]
```diff
@@ -184,95 +190,121 @@ def benchmark_train_and_valid(base_dir, train_dataset, valid_dataset, tasks,
...
+    # Loading hyper parameters
+    batch_size = hyper_parameters['batch_size']
+    nb_epoch = hyper_parameters['nb_epoch']    
     learning_rate = hyper_parameters['learning_rate']
     n_filters = hyper_parameters['n_filters']
     n_fully_connected_nodes = hyper_parameters['n_fully_connected_nodes']
-    batch_size = hyper_parameters['batch_size']
-    nb_epoch = hyper_parameters['nb_epoch']
-    
+    seed = hyper_parameters['seed']
```
Comment (rbharath) [inline examples/benchmark.py]: I think `seed` shouldn't be a hyperparameter. It should just be a number like (123) that ensures that all users run tests under the same conditions.

**Comment by rbharath** [comment]
@miaecle Thanks for these changes! Adding on the stddevs as hyperparameters is a good idea since they can be critical for getting good performance. I think during the merge, the `seed` parameters were removed, so I've added a few comments noting where they need to be added back in.

**Comment by miaecle** [comment]
@rbharath Thanks for the review! I have added the seed as a default parameter so as to make it consistent for all models, let me know if any other changes are needed!

**Comment by rbharath** [comment]
@miaecle Thanks, looks good! Going to merge in.
